src/gie.c

```diff
@@ -146,7 +146,7 @@ static ffio *ffio_create (const char **tags, size_t n_tags, size_t max_record_si
 
 static const char *gie_tags[] = {
     "<gie>", "operation", "accept", "expect", "roundtrip", "banner", "verbose",
-    "direction", "tolerance", "builtins", "echo", "</gie>"
+    "direction", "tolerance", "skiperror", "builtins", "echo", "</gie>"
```

Comment:
Suggest "ignore" rather than "skiperror": Seems linguistically more in line with "accept" & "expect"